src/services/application.js

```diff
@@ -28,6 +33,17 @@ export class CompanyApplicationEmailAlreadyInUse extends Error {
     }
 }
 
+export class CompanyApplicationAlreadyValidated extends Error {
+    constructor(msg) {
+        super(msg);
+    }
+}
+export class CompanyApplicationUnverified extends Error {
+    constructor(msg) {
+        super(msg);
+    }
+}
+
```

Comment:
It seems to me that these classes are not adding any additional information. Apparently, we were using it before but I don't see any reason to use them, as they are constructed in the same way as an Error. Are we veryfing the type of class somewhere above (to see if it's not unknown error or something)?

Reply:
Yes we use them to distinguish between errors